roles/setup_netobserv_stack/tasks/main.yml

```
@@ -0,0 +1,21 @@
+---
+- name: Install the NetObservability stack
+  when:
+    - setup_netobserv_stack_action | default(dsetup_netobserv_stack_action) == 'install'
```

If the default values are not prefixed with `d` this could be changed to
```suggestion
    - setup_netobserv_stack_action == "install"
```

---

Done